src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -143,6 +166,14 @@ class OverviewRoute extends React.Component<OverviewRouteProps, State> {
 
         {!hideMainOverviewSection && <Spacer mb={4} />}
 
+        {showCollectionsRail && ( // TODO: remove after CollectionsRail a/b test
+          <div>
+            <Separator mb={3} />
+            <ArtistCollectionsRail />
+            <Spacer mb={3} />
+          </div>
```

Comment:
I remember learning a cool shorthand for `<div>` from @jonallured, I think. Unless you're passing properties to the tag, you can use:

```js
<>
  <Separator mb={3} />
  <ArtistCollectionsRail />
  <Spacer mb={3} />
</>
```

Reply:
Unfortunately the fragment doesn't seem to work well as a wrapper for the carousel

Reply:
Oops, misunderstanding somewhere, the short hand syntax you're showing here would result in a Fragment, see also:

https://reactjs.org/docs/fragments.html#short-syntax

Looks like <div></div> is what @eessex is after here.

Reply:
Ahhh, gotcha. :+1: